step-plans/step-plans-base-artefacts/src/main/java/step/artefacts/handlers/asserts/AbstractOperatorHandler.java

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+ * Copyright (C) 2020, exense GmbH
+ *
+ * This file is part of STEP
+ *
+ * STEP is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * STEP is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with STEP.  If not, see <http://www.gnu.org/licenses/>.
+ ******************************************************************************/
+package step.artefacts.handlers.asserts;
+
+import java.math.BigDecimal;
+
+public abstract class AbstractOperatorHandler implements AssertOperatorHandler {
+    protected String not(boolean negate) {
+        return negate ? " not " : " ";
+    }
+
+    protected boolean isNumber(Object value) {
+        return value instanceof Number;
+    }
+
+    protected boolean isBoolean(Object value) {
+        return value instanceof Boolean;
+    }
+
+    protected boolean isString(Object value) {
+        return value instanceof String;
+    }
+
+    protected boolean isNull(Object value) {
+        return value == null;
+    }
+
+    protected int compareNumeric(Object actual, String expectedValueString) {
+        if (actual instanceof Integer) {
+            return ((Integer) actual).compareTo(Integer.valueOf(expectedValueString));
+        } else if (actual instanceof Long) {
+            return ((Long) actual).compareTo(Long.valueOf(expectedValueString));
+        } else if (actual instanceof Double) {
```

Comment:
@iegorov777 I've re-executed the tests with coverage. It seem that Long, Double and Float are still not covered by the JUnit tests of AssertHandlerTest

Reply:
Double and Float were covered by separate tests (GreaterThanOperatorHandlerTest and LessThanOperatorHandlerTest), because it is hard to test them under AssertHandlerTest (when we use double/floats in json output data in this test, these values are automatically parsed as BigDecimals by javax.json, so we only can get BigDecimals in AssertHandler there).

FYI: In my last commit I've exteded GreaterThanOperatorHandlerTest and LessThanOperatorHandlerTest to test various types of expected values as well as types of actual values.

Reply:
@iegorov777 My bad. I haven't seen these 2 tests. Thanks for the additional tests